CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+## Contributors guidelines
+ 
+### Getting started
+To start contributing to the Onfido Web SDK, clone this repository and run the following commands:
+ 
+- `npm install` to install the project dependencies
+- `npm run dev` to generate the build and run the dev server
+ 
+The SDK will be running on https://localhost:8080.
+ 
+Note: The SDK runs on `https`, which is needed to support the camera functionality.
+ 
+You can also run it using `docker` and `docker-compose` by running the following command: `docker-compose -f docker-compose.dev.yml up`.
+ 
+### Testing
+The Web SDK has a large coverage of UI tests. To set up your environment, please refer to the [testing guidelines](./test/TESTING_GUIDELINES.md).
+Once your environment is ready, you can run UI tests by running `npm run build:test && npm run travis && npm run test:ui`.
+The Web SDK has partial coverage of unit tests that can be executed with `npm run test`.
+You can also run a type checker and linter by running `npm run check`.
+You should also manually test any change in all the supported browsers, both on desktop and mobile. To facilitate this process, every time a new pull request is created, a new demo app link will be generated. If you don't have enough devices to test your changes, you can perform manual tests on different devices on Browserstack.
+ 
+### Contributing
+When creating a new branch, contributors should use the following convention `{task-type}/{task-description}-{ticket-number}`.
+The most used task types are `feature`, `fix` or `improvement`. The ticket number is optional and is only expected from internal contributors.
+ 
+For more details, check out the [pull request checklist](./.github/PULL_REQUEST_TEMPLATE.md).
+ 
+In order to merge your pull request, you should get an approval from 2 Onfido team members. 
+ 
+The current pull request checks are
+- Bundlesize - A tool that monitors the SDK size increase
+- Travis CI - Our continuous integration tool
+ 
+At build time, our CI runs the following tests:
+- UI
+- Unit
+- Type checker
+- Linter
+- Dependencies vulnerability detection
+ 
+All of the above must have passed in order for the build to succeed. Please note that the CI relies on encrypted variables to run tests therefore the build will only be executed if the pull request has been created by an internal contributor and it will always fail for pull requests from external contributors.
+For internal contributors, all the PR checks must pass. 
+ 
+### Internationalization
+ 
+The SDK supports several languages. To pull the translations from our localisation service run `npm run lokalise:download`.
```

Comment:
Would it work for an external client's Lokalise also? Else we might need to be a bit more explicit about this being only applicable to Onfido's localisation strings.